lib/matrix_sdk/protocols/cs.rb

```diff
@@ -767,6 +767,30 @@ def get_room_state_all(room_id, **params)
     request(:get, :client_r0, "/rooms/#{room_id}/state", query: query)
   end
 
+  # Retrieves number of events that happened just before and after the specified event
+  #
+  # @param room_id [MXID,String] The room to get events from.
+  # @param event_id [MXID,String] The event to get context around.
+  # @option limit [Integer] (10) The limit of messages to retrieve
```

Comment:
This should probably be `@option params [Integer] :limit (10)`, the first part - the variable - is the source of the option, then you have the type, then the name of the option `:limit`, and then the default value

Ref: https://rubydoc.info/gems/yard/file/docs/Tags.md#option

Reply:
ahh yeah thats my bad. I didnt fixup the definition after changing this from an @param to an @option